core/src/main/kotlin/renderers/html/htmlPreprocessors.kt

```diff
@@ -43,7 +43,7 @@ object NavigationPageInstaller : PageNodeTransformer {
 
     private fun visit(page: ContentPage): NavigationNode = NavigationNode(
         page.name,
-        page.dri,
+        page.dri.first(),
```

Comment:
Not sure about this (you don't change pages in this PR)

Reply:
Some leftover after fixing the build proccess

Reply:
Will remove